src/platform/forms-system/src/js/patterns/array-builder/ArrayBuilderSummaryPage.jsx

```diff
@@ -16,21 +16,19 @@ import {
   isDeepEmpty,
 } from './helpers';
 
-const SuccessAlert = forwardRef(
-  ({ nounSingular, index, onDismiss, text }, ref) => (
-    <div className="vads-u-margin-top--2" ref={ref}>
-      <VaAlert
-        onCloseEvent={onDismiss}
-        closeable
-        name={`${nounSingular}_${index}`}
-        status="success"
-        closeBtnAriaLabel="Close notification"
-        uswds
-      >
-        {text}
-      </VaAlert>
-    </div>
-  ),
+const SuccessAlert = ({ nounSingular, index, onDismiss, text }) => (
+  <div className="vads-u-margin-top--2">
```

Comment:
Always glad to have `ref`s removed!!
EDIT: Ah, I see it was just moved. Well, still, seems better how you have it here 😄